crates/hyperdrive-wrappers/Cargo.toml

```diff
@@ -10,19 +10,26 @@ build = "build.rs"
 
 # External dependencies
 ethers = "2.0.8"
+ethers-solc = "2.0.8"
```

Comment:
Why is this a runtime dependency now? It should still only be used in `build.rs`

Reply:
Nvm, I see where it is used